examples/dynamo_integration/scripts/plot_throughput_histograms_per_request.py

```
@@ -0,0 +1,861 @@
+                    num_str = item_id.split("_")[-1]
+                    example_num = int(num_str)
+                except (ValueError, IndexError):
+                    pass
+            
+            scores_by_example[example_num] = {
+                "id": item_id,
+                "score": score,
+                "reasoning": reasoning
+            }
+        
+        return scores_by_example
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)